mos-platform/nes/rompoke/rompoke-table.ld

```diff
@@ -0,0 +1,1028 @@
+/* Automatically generated by generate-rompoke-table-linkscript.lua. */
+SECTIONS {
```

Comment:
A little generated comment here to describe how each block works and the purpose would go a long way towards readability.

Also, this is a very nice hack; it depends only on well-trodden paths, and it thus seems likely to be maintainable.

Reply:
The comment should be taken care of now.